dtos/http_response.go

```diff
@@ -0,0 +1,39 @@
+package dtos
+
+type ErrorResponse struct {
+	Code    int         `json:"code"`
+	Message string      `json:"message"`
+	Detail  interface{} `json:"detail"`
```

Comment:
why we need Details here, as long we have message..?

Reply:
if we were to send some other data like a json object or something